ARCs/arc-0027.md

```diff
@@ -0,0 +1,1611 @@
+---
+arc: 27
+title: Web Extension Wallet Interface
```

Comment:
What do you think of keeping this agnostic to the platform? We could just say Wallet Interface for instance